lang/en/cpdlogbook.php

```diff
@@ -45,3 +45,5 @@
 $string['periods'] = 'Periods';
 $string['startdate'] = 'Start date';
 $string['enddate'] = 'End date';
+$string['startendinvalid'] = 'Start date after end date';
+$string['overlap'] = 'This overlaps with an existing period';
```

Comment:
This pr looks good to land

At some point its also good to sort the lang strings and keep the sorted / organised which helps reduce redundant strings